data/json/items/comestibles.json

```diff
@@ -4238,7 +4262,7 @@
     "material": "veggy",
```

Comment:
hickory nut ambrosia includes maple syrup with a freezing point of -70F, so it should probably have a lower freezing point but the exact value I don't know.